shared/profile/user/weboftrust/index.tsx

```diff
@@ -1,68 +1,106 @@
+import {WotStatusType} from '../../../constants/types/rpc-gen'
 import * as Kb from '../../../common-adapters'
 import * as Styles from '../../../styles'
+import {formatTimeRelativeToNow} from '../../../util/timestamp'
 
 type Props = {
   attestation: string
   attestingUser: string
-  dateString: string
   onAccept?: () => void
   onHide?: () => void
   onReject?: () => void
-  pending: boolean
-  verificationType: Types.WebOfTrustVerificationType
+  reactWaitingKey: string
+  status: WotStatusType
+  userIsYou: boolean
+  username: string
+  verificationType: WebOfTrustVerificationType
+  vouchedAt: number
 }
 
-const WebOfTrust = (props: Props) => (
-  <>
-    <Kb.Box2 direction="horizontal" fullWidth={true}>
-      <Kb.Box2 direction="vertical" style={styles.avatarContainer} centerChildren={true}>
-        <Kb.Avatar
-          size={96}
-          username={props.attestingUser}
-          style={styles.avatar}
-          showFollowingStatus={false}
-        />
-      </Kb.Box2>
-      <Kb.Box2 direction="vertical" fullWidth={true} style={styles.textContainer} centerChildren={false}>
-        <Kb.Text style={styles.attestationText} type="Body">
-          {props.attestation}
-        </Kb.Text>
-        <Kb.Box2 direction="vertical" style={styles.signatureBox} centerChildren={false} fullWidth={true}>
-          <Kb.Box2
-            direction="horizontal"
-            gap="xxtiny"
-            centerChildren={true}
-            style={{alignSelf: 'flex-start'}}
-          >
-            <Kb.Icon color={Styles.globalColors.blue} type="iconfont-proof-good" sizeType="Small" />
-            <Kb.Text type="BodySmall">signed by </Kb.Text>
-            <Kb.ConnectedUsernames
-              type={Styles.isMobile ? 'BodySmallBold' : 'BodyBold'}
-              usernames={props.attestingUser}
-              colorBroken={true}
-              colorFollowing={true}
-              style={styles.username}
-            />
-            <Kb.Text type="BodySmall">{props.dateString}</Kb.Text>
+const entry = (props: Props) => {
+  return (
+    <>
+      <Kb.Box2 direction="horizontal" fullWidth={true}>
+        <Kb.Box2 direction="vertical" style={styles.avatarContainer} centerChildren={true}>
+          <Kb.Avatar
+            size={96}
+            username={props.attestingUser}
+            style={styles.avatar}
+            showFollowingStatus={false}
+          />
+        </Kb.Box2>
+        <Kb.Box2 direction="vertical" fullWidth={true} style={styles.textContainer} centerChildren={false}>
+          <Kb.Text style={styles.attestationText} type="Body">
+            {props.attestation}
+          </Kb.Text>
+          <Kb.Box2 direction="vertical" style={styles.signatureBox} centerChildren={false} fullWidth={true}>
+            <Kb.Box2
+              direction="horizontal"
+              gap="xxtiny"
+              centerChildren={true}
+              style={{alignSelf: 'flex-start'}}
```

Comment:
best to try to get every style in a `styles.someStyle` entry, even if it's a short one